README.md

```diff
@@ -44,7 +44,7 @@ using PythonCall, BraketSimulator
 
 ## Usage Notes
 
-If you want to use this package **from Python**, see the Python wrapper package [`amazon-braket-simulator-v2-python`](), which provides a Python interaction layer and will install Julia, this package and other Julia dependencies in a sandboxed virtual environment for you.
+If you want to use this package **from Python**, see the Python wrapper package [`amazon-braket-simulator-v2-python`](https://github.com/amazon-braket/amazon-braket-simulator-v2-python), which provides a Python interaction layer and will install Julia, this package and other Julia dependencies in a sandboxed virtual environment for you.
```

Comment:
Will everything be sandboxed?

Reply:
If the user already has a `CondaPkg.toml` or `juliapkg.json` running around somewhere `PythonCall.jl`/`juliacall` will try to reconcile all the compat entries. I'll try to find some good phrasing to make a note of that.

Reply:
I was more worried about giving people the impression that we'll create a virtual env for their python dependencies then the nuance of juliapkg. But I think I was nitpicking.